include/rdma/fabric.h

```diff
@@ -342,6 +342,8 @@ struct fi_domain_attr {
 	size_t			mr_iov_limit;
 	uint64_t		caps;
 	uint64_t		mode;
+	uint8_t			*default_auth_key;
+	size_t 			default_auth_keylen;
```

Comment:
I would just match the names for the ep_attr, and remove 'default' from the prefix.